ts/private/ts_project.bzl

```diff
@@ -181,7 +181,7 @@ See https://github.com/aspect-build/rules_ts/issues/361 for more details.
 
     typings_srcs = [s for s in srcs_inputs if _lib.is_typings_src(s.path)]
 
-    if len(js_outs) + len(typings_outs) < 1:
+    if len(js_outs) + len(typings_outs) < 1 and not ctx.attr.no_emit:
```

Comment:
the code inside this branch mentions issues/88 so it needs an update also

Reply:
We can also just delete this entire block now. I'm just not sure if we want to make such a change at this time? Currently this PR has is entirely opt-in with the `no_emit` flag...

Reply:
so then let's just be conservative and update the text, since issue 88 is being closed here

Reply:
done